editoast/src/views/timetable.rs

```diff
@@ -326,6 +334,17 @@ async fn post_paced_train(
         return Err(TimetableError::NotFound { timetable_id }.into());
     }
 
+    // Check duplicated paced train exceptions keys
+    for paced_train in &paced_trains {
+        if let Some(duplicated_key) = paced_train.find_first_duplicated_exception_key() {
+            return Err(TimetableError::ExceptionKeyDuplicate {
+                exception_key: duplicated_key.to_string(),
+                train_name: paced_train.train_schedule_base.train_name.clone(),
+            }
+            .into());
+        }
+    }
```

Comment:
We could handle this validator in the deserialization of the object.
So we do not forget to perform this check.